File: src/main/java/data/astro/AstroCalculator.java
package data.astro;

import java.util.Calendar;
import java.util.GregorianCalendar;

import utils.AstroUtils;

public class AstroCalculator {
    public String calc(int date, double lat, double lon, double localZone) {
        Calendar calendar = GregorianCalendar.getInstance();
        calendar.set(2020, Calendar.JANUARY, date);
        String sunRise = AstroUtils.getSunRise(
                calendar.getTimeInMillis(), lat, lon, localZone
        );
        String sunSet = AstroUtils.getSunSet(
                calendar.getTimeInMillis(), lat, lon, localZone
        );
        StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.append("calendar:").append(calendar.getTime());
        stringBuilder.append("\nsunRise:").append(sunRise);
        stringBuilder.append("\nsunSet:").append(sunSet);
        return stringBuilder.toString();
    }
}

File: src/main/java/data/astro/AstroCalculator.kt
package data.astro

import utils.AstroUtils

class AstroCalculator {
    fun calc(timeInMillis: Long, lat: Double, lon: Double, localZone: Double = 0.0): String {
        val sunRise = AstroUtils.getSunRise(timeInMillis, lat, lon, localZone)
        val sunSet = AstroUtils.getSunSet(timeInMillis, lat, lon, localZone)
        return """
            sunRise:$sunRise
            sunSet:$sunSet
            """.trimIndent()
    }
}